### ISSUE TITLE
NOWEB - add poll button in api

### ISSUE THREAD

**Issue by allburov** [allburov/whatsapp-http-api#131]
From @mycodemm 

--- 

Dear Team,

add poll button in API, as poll option is already available in lib, it is a good alternative for buttons. also, it is very useful for companies to get reviews or votes.

can create multiple poll options in one message, can send a text with poll, Text+Media+Poll, Text+Docs+Poll.

One more use of poll, if any receiver click on poll then it creates a report of interested user like
if you want to receive this message poll: yes or no so its a small alternative for a button also.

**Comment by jgabriellima** [comment]
Yes, please! I really need this functionality. I'm using the plus version!

**Comment by allburov** [comment]
WEBJS so far doesn't support polls :(

**Comment by allburov** [comment]
Hi! Poll is available in NOWEB engine, here's documentation https://waha.devlike.pro/docs/how-to/polls/

Overall, you need to send `POST /api/sendPoll`  request 
```json
{
  "session": "default",
  "chatId": "123123123@c.us",
  "poll": {
    "name": "How are you?",
    "options": [
      "Awesome!",
      "Good!",
      "Not bad!"
    ],
    "multipleAnswers": false
  }
}
```

After that you'll process `poll.vote` and `poll.vote.failed` events that you get in webhooks

**Comment by allburov** [comment]
* Released in WAHA 2023.08.29 Plus version (both x86 and arm)
* will release it in Core in September
